core/lib/zksync_core/src/eth_sender/tests.rs

```diff
@@ -477,7 +510,14 @@ async fn three_scenarios() -> anyhow::Result<()> {
 #[tokio::test]
 async fn failed_eth_tx() {
     let connection_pool = ConnectionPool::test_pool().await;
-    let mut tester = EthSenderTester::new(connection_pool.clone(), vec![100; 100], false).await;
+    let l1_batch_commit_data_generator = Arc::new(RollupModeL1BatchCommitDataGenerator {});
```

Comment:
Dumb question: here, and in quite a few other tests you pass "hard-coded" `RollupModeL1BatchCommitDataGenerator` as a parameter throughout the test. Do you plan to adapt these tests to run with the other generator as well? If not, it could be inlined down the call chain (e.g., into the `Aggregator::new()` constructor in `EthSenderTester::new()` in this case), so that the changeset is smaller and the PR is easier to review. In this particular case, a comment right above `Aggregator::new()` says "Aggregator - unused", so I'm not sure having a generator as an argument would even make sense.

Reply:
They have been merged already, resolving this conversation